docker-compose.yml

```diff
@@ -10,9 +8,16 @@ app:
     - AWS_ACCESS_KEY
     - AWS_SECRET_KEY
   links:
-    - rabbitmq
-    - datomic
+    - wildfly
 rabbitmq:
   image: rabbitmq:3.5.0-management
-datomic:
-  image: quay.io/democracyworks/datomic-tx:0.9.5153
```

Comment:
You would link the transactor container in docker-compose.yml and then set USPS_DATOMIC_URI like this:

```yaml
wildfly:
  image: ...
  links:
    - datomic
  environment:
    USPS_DATOMIC_URI: datomic:dev://datomic:4334/usps-processor
datomic:
  image: quay.io/democracyworks/datomic-tx:0.9.5153
```

It can be handy in dev to have Datomic data persist across app restarts, which docker-compose should do by default since it just stops and starts the containers unless you explicitly ask it to delete them.